_data/navbar.yml

```diff
@@ -31,6 +31,7 @@
       - xen-api/metrics.md
       - xen-api/memory.md
       - xen-api/xencenter.md
+        # - profiling/coverage.md
```

Comment:
It is not needed, because you are using `design_doc: true`, which causes it to show up in the table of design docs.